x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java

```diff
@@ -113,6 +119,20 @@ private void stopAnalyticsAndForceStopOnError() {
         }
     }
 
+    protected static void createDataStreamAndTemplate(String dataStreamName, String timeField, String mapping) throws IOException {
```

Comment:
Could we move this to `MlNativeIntegTestCase` and then also reuse it in `DatafeedJobsIT`?